eslint.config.js

```diff
@@ -5,26 +5,27 @@ import globals from "globals";
 export default [
     {
         ignores: [
-            "tests/fixtures/",
-            "dist/",
-            "coverage/",
-            "tools/create-test-example.js"
+            "**/tests/fixtures/",
+            "**/dist/",
+            "**/coverage/",
+            "packages/espree/tools/create-test-example.js"
         ]
     },
     ...eslintConfigESLint,
     eslintConfigESLintFormatting,
     {
-        files: ["tests/lib/**"],
+        files: ["**/tests/lib/**"],
         languageOptions: {
             globals: {
                 ...globals.mocha
             }
         }
     },
     {
-        files: ["tools/**"],
+        files: ["**/tools/**"],
         rules: {
             "no-console": "off",
+            "no-tabs": "off",
```

Comment:
Since this repo is still using eslint formatting rules, I think we should keep this rule enabled for now and replace those tabs with spaces.